vulnfeeds/cves/versions_test.go

```diff
@@ -41,3 +41,47 @@ func TestParseCPE(t *testing.T) {
 		}
 	}
 }
+
+func TestExtractGitHubCommit(t *testing.T) {
+	tests := []struct {
+		description       string
+		inputLink         string
+		expectedFixCommit *FixCommit
+	}{
+		{
+			description: "Valid GitHub commit URL",
+			inputLink:   "https://github.com/google/osv/commit/cd4e934d0527e5010e373e7fed54ef5daefba2f5",
+			expectedFixCommit: &FixCommit{
+				Repo:   "https://github.com/google/osv",
+				Commit: "cd4e934d0527e5010e373e7fed54ef5daefba2f5",
+			},
+		},
+		{
+			description:       "Unsupported GitHub PR URL",
+			inputLink:         "https://github.com/google/osv/pull/123",
+			expectedFixCommit: nil,
+		},
+		{
+			description:       "Unsupported GitHub tag URL",
+			inputLink:         "https://github.com/google/osv.dev/releases/tag/v0.0.14",
```

Comment:
just a note: we should make these (pull requests + tag links) work at some point (by turning them into a commit hash).

Reply:
Yeah I wanted to capture some of the things we'd like to have work in the future.